Source/ACE.Entity/AceObject.cs

```diff
@@ -11,7 +11,7 @@ namespace ACE.Entity
     [DbTable("ace_object")]
     public class AceObject : ICreatureStats, ICloneable
     {
-        public const ushort WEENIE_MAX = 0xFFFF;
+        public const uint WEENIE_MAX = 199999;
```

Comment:
In theory, the maximum it could be set to would be what ever number you choose to start actual spawned instances of ace_objects at minus 1. For every weenie you have there is an entry in both the ace_weenie_class table and ace_object both ids being used for their index, so to prevent overlap of an instance a ACE generated instance or pulled from PCAP instance you want to pick a limit.

Likely we could change 199999 to 19999999 to match the true amount of space I'm building in ACE-World for any newly generated weenies, but of course its highly unlikely we'd use anywhere near that much.

The number is "compressed" by a function call so while the client always reads it as a ushort as I understand it, it gets converted because otherwise the client was always limited to less than 32K weenies and we know they went to the 50K+ range.